src/ui/QuerySuggestPreview/QuerySuggestPreview.ts

```diff
@@ -22,6 +22,21 @@ export interface IQuerySuggestPreview {
   executeQueryDelay?: number;
 }
 
+/**
+ * This component adds previews of the top query results matching the currently focused query suggestion in the search box.
+ *
+ * As such, this component only works when an [`Omnibox`]{@link Omnibox} is present in the search interface.
+ *
+ * Moreover, this component can use a [result template](https://docs.coveo.com/en/413/) in its markup configuration to render previews with.
+ *
+ * **Example**
+ * This component works independently of its location in the DOM and can be added like such:
+ * ```html
+ *   <div class="CoveoQuerySuggestPreview"></div>
+ * ```
+ *
+ * See [Providing Query Suggestion Result Previews](https://docs.coveo.com/en/340/#providing-query-suggestion-result-previews).
```

Comment:
Please change the link's label / anchor to:

Rendering Query Suggestion Result Previews / #rendering-query-suggestion-result-previews

Reply:
Alright, I'm changing the label and anchor.